### PR TITLE
Feature/python pre commits

### PR DESCRIPTION
**Pull request by cathex-ryan-deamer**

### CODE REVIEW

File: pre_commit_python_hooks/check_latest_in_module_source.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    args = parser.parse_args(argv)
+    # print(args)
+
+    for arg in args.filenames:
+        with open(arg, "r+") as file:
+            lines = file.readlines()
+
+            for index, line in enumerate(lines):
+
+                if "#ignore" in line:
```
Comment (cathex-matt) [inline pre_commit_python_hooks/check_latest_in_module_source.py]: Could we use `# tf:ignore` to make sure we understand the perimeter of the instruction

**Review by cathex-matt — APPROVED** [review]
approved with a comment

File: pre_commit_python_hooks/check_latest_in_module_source.py (hunk 1 above)
Comment (cathex-matt) [reply pre_commit_python_hooks/check_latest_in_module_source.py]: Maybe it's worth adding the pragma in the README in a section which does not exist yet like `Hooks Documentation`

Comment (cathex-ryan-deamer) [reply pre_commit_python_hooks/check_latest_in_module_source.py]: ignore has been update to # tf:ignore

**Pull request merged**